docs/migrating-from-onenote.md

```diff
@@ -0,0 +1,20 @@
+# Migrating from OneNote
+
+This guide mostly duplicates the instructions at the repo for the PowerShell [script](https://github.com/nixsee/ConvertOneNote2MarkDown).
+
+## Usage
+
+1. Start the OneNote application. All notebooks currently loaded in OneNote will be converted
+2. It is advised that you install Onetastic and the attached macro, which will automatically expand any collapsed paragraphs in the notebook. They won't be exported otherwise.
+    * To install the macro, click the New Macro Button within the Onetastic Toolbar and then select File -> Import and select the .xml macro included in the release.
+    * Run the macro for each Notebook that is open
+3. It is highly recommended that you use VS Code, and its embedded Powershell terminal, as this allows you to edit and run the script, as well as check the results of the .md output all in one window.
+4. Whatever you choose, open a PowerShell terminal and navigate to the folder containing the script and run it.
```

Comment:
Perhaps we should include instructions on how to navigate with cd commands? Or how to add a folder in VS Code and right click to open it in terminal?

![image](https://user-images.githubusercontent.com/24255237/89111899-8db56380-d418-11ea-8309-01c1e4a62e96.png)

Reply:
Also, "run it" is done through typing ".\ConvertOnenote2Markdown-v2"